Sources/SwiftDocC/SourceRepository/SourceRepository.swift

```diff
@@ -89,4 +89,16 @@ public extension SourceRepository {
             formatLineNumber: { line in "lines-\(line)" }
         )
     }
+    
+    /// Creates a source repository hosted by the device's filesystem.
+    ///
+    /// Use this source repository to format `file://` links to files on the
+    /// device where documentation is being presented.
+    static func localFilesystem() -> SourceRepository {
+        SourceRepository(
+            checkoutPath: "",
+            sourceServiceBaseURL: URL(string: "file:///")!,
```

Comment:
Is this meant to refer to the root because the other file paths are absolute? If so, I think the `fileURLWithPath` initializer would make signal this intention better:

```suggestion
            sourceServiceBaseURL: URL(fileURLWithPath: "/"),
```

Reply:
Ah that's a good, point. Fixed